README.md

```diff
@@ -127,6 +127,7 @@ Options:
 - **`--debug`**: Run in debug mode. When set, it does the following:
   - uses [debug](https://github.com/visionmedia/debug) internally to log additional information about staged files, commands being executed, location of binaries, etc. Debug logs, which are automatically enabled by passing the flag, can also be enabled by setting the environment variable `$DEBUG` to `lint-staged*`.
   - uses [`verbose` renderer](https://github.com/SamVerschueren/listr-verbose-renderer) for `listr`; this causes serial, uncoloured output to the terminal, instead of the default (beautified, dynamic) output.
+    (the [`verbose` renderer](https://github.com/SamVerschueren/listr-verbose-renderer) can also be activated by setting the `TERM=dumb` or `NODE_ENV=test` environment variables)
```

Comment:
We're actually using `listr2` now, so let's link here instead: https://listr2.kilic.dev/renderers/verbose-renderer